src/modules/dashboard/Dashboard.vue

```diff
@@ -90,26 +147,46 @@
   text-align: left;
   width: 50%;
 }
+.viewMoreTestBtn{
+  border:1px solid #dc3545;
+  background-color:white;
+  height:40px;
+  width:140px;
+  padding:0px;
+  padding-bottom:2px;
+}
+.viewMoreTestBtn:focus{
+  box-shadow:none !important;
+  outline:none !important;
+}
+.viewMoreTestBtn:hover{
+  background-color: #dc3545;
+  color:white;
+  box-shadow: 0px 0px 30px #bfbfbf;
+}
 </style>
 <script>
 import QrcodeVue from 'qrcode.vue'
 import AUTH from 'src/services/auth'
 import COMMON from 'src/common.js'
 import CONFIG from 'src/config.js'
 import ComplaintProperty from './Complaint.js'
+import ROUTER from 'src/router'
 export default{
   mounted(){
     this.retrieve()
     if(this.userTypes.indexOf(this.user.type) > -1) {
       this.dashType = 'USER'
     }
+    window.addEventListener('resize', this.QRResizer)
```

Comment:
please add api for removing the event listener on destroyed/unmount this is to avoid memory leaks especially for older browsers

Reply:
@keyrwinz so ako rang e condition ang pag addEventListener() ? pwede ako rang e condition; 

export default{
  mounted(){
    this.retrieve()
    if(this.userTypes.indexOf(this.user.type) > -1) {
      this.dashType = 'USER'
    }
    // to set the size of the QRCode based on screen size
    if(window.innerWidth <= 842 || window.innerWidth >= 690){
        this.qrSize = 180
     }
     if(window.innerWidth > 575 || window.innerWidth <= 689){
        this.qrSize = 160
     }
     if(window.innerWidth < 576 || window.innerWidth > 1000){
        this.qrSize = 220
     }
     // ends here

Reply:
sa vuejs naa ty mounted() nga mu trigger if na mount na ang component, in contrast ani naa sd ty destroyed() nga mu trigger sd ig unmount sa component. so inside sa destroyed() you can put didto ang pg remove sa event listener.

e.g.
destroyed() {
      window.removeEventListener('resize', this.QRResizer)
}

you can explore more information here about vuejs lifecycle hooks
https://vuejs.org/v2/guide/instance.html

Reply:
@keyrwinz yes ako ra nang e implement

Reply:
@BargasoRC ok nani sya? arun ma resolve nani

Reply:
actually ako na nang gi remove tanan kay lahi man ang design sa admin dashboard ok na na

Reply:
@keyrwinz  wala pa nako gi work balik ang admin dashboard kay gi prioritize nako ang LGU and brgy dashboard

Reply:
@keyrwinz pero human na man ang akong LGU and brgy dashboard wala pa lang na merge

Reply:
ok noted